### ISSUE TITLE
Enhancement: Improve accessibility for sight impared users

### ISSUE THREAD

**Issue by julien-gibson** [geoffhumphrey/brewcompetitiononlineentry#1090]
Version: 2.1.17

Description of Issue: Site compatibility with screen readers is pretty sketchy.  There are problems around pop up dialogs and also action links not being read.

**Comment by geoffhumphrey** [comment]
You are absolutely correct. A greater effort will be made to make BCOE&M fully compatible with screen readers for the 3.0.0 conversion release.
